windows/README.md

```diff
@@ -0,0 +1,77 @@
+# The Flutter Windows Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on the Windows desktop.
+
+This README assumes you have already read [the top level README](../README.md), which contains important information and setup common to all platforms.
+
+## Flutter Location
+
+Contrary to [the top level README](../README.md) Windows cannot be run using a `.flutter_location_config` file and a Flutter tree must be located in the same parent directory as this repo like so:
+
+```
+<parent dir>
+  ├─ flutter (from http://github.com/flutter/flutter)
+  └─ flutter-desktop-embedding (from https://github.com/google/flutter-desktop-embedding)
+```
+
+## Adding Windows as a Target Platform
+
+By default flutter will throw an unknown target platform error if run on Windows. Windows can be added by editing `\path\to\flutter\packages\flutter\lib\src\foundation\platform.dart`.
+
+Change the code around line 37 from this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux) {
+  result = TargetPlatform.android;
+}
+```
+
+to this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux || Platform.isWindows) {
+  result = TargetPlatform.android;
+}
+```
+
+The flutter app you are trying to run must be rebuilt before it can be run on windows.
+
+```
+> flutter build bundle
+```
+
+## Using the Library
+
+Build the GLFW Library project in Visual Studio into a static or dynamic library, then link `flutter_embedder.lib` and include `embedder.h` into your binary. Also ensure that the `flutter_engine.dll`, and if using a dynamic library `flutter_embedder.dll`, are in valid DLL include paths.
+
+The output files are located in `bin\x64\$(Configuration)\GLFW Library\`.
+
+## Example Application
+
+The application under `GLFW Example\` shows an example application using the library.
+
+The example application must be built before running the windows executable as Visual Studio doesn't automatically do this.
+
+```
+> flutter build bundle
+```
+
+You should be able to build the GLFW Example project in Visual Studio and have access to `GLFW Example.exe` located in `bin\x64\$(Configuration)\GLFW Example\`.
+
+The resulting binary expects to be run from this directory like so:
+
+```
+> ".\bin\x64\$(Configuration)\GLFW Example\GLFW Example.exe"
+```
+
+e.g:
+
+```
+> ".\bin\x64\Debug .dll\GLFW Example\GLFW Example.exe"
+```
+
+or you can use Visual Studio's inbuilt debugger to build the library and run the example application automatically.
```

Comment:
Does this work for you? I can successfully run it from the command line, but launching it with the debugger gives me the exceptions that have been discussed by others in the bug and mailing list.

Reply:
I'm still curious about this; you're able to run from within VS without exceptions?

Reply:
I am able to run completely fine. Are the exceptions from the compiler, linker, script failure, flutter engine or something else. If you can run without the debugger then I would think it would be your Visual Studio installation. Can you confirm with another project that Visual Studio is working correctly?